Algorithms/DynamicProgramming(DP)/Kadane's Algorithm.cpp

```diff
@@ -0,0 +1,36 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+int kadanes(int arr[],int a)
+{
+    int current=arr[0];
+    int global=arr[0];
+    for(int i=1;i<a;i++)
+    {
+        if(arr[i]>current+arr[i])
+        current=arr[i];
+        else
+        current=(current+arr[i]);
+        
+        if(current>global)
+        global=current;
+    }
+    
+    return global;
+}
+
+int main() {
+	int t;
+	cin>>t;;
+	while(t--)
```

Comment:
please avoid using single letter variables and replace with sensible names

Reply:
Changes done as per the requirements.

Reply:
Please review.